lib/experimental/new_embed.dart

```diff
@@ -105,12 +112,12 @@ class NewEmbed {
           ..theme = 'elegant'
           ..mode = 'dart'
           ..showLineNumbers = true;
-
     userCodeEditor.document.onChange.listen(_performAnalysis);
 
     testEditor = editorFactory.createFromElement(querySelector('#test-editor'))
       ..theme = 'elegant'
       ..mode = 'dart'
+      ..readOnly = true
```

Comment:
We've been leaving the test editor writeable in the short-term as a convenience so that Kathy (and now Jon) can use the live UI to write/test gists as they're drafting their codelabs.

Reply:
I'll remove this (and add a todo to make it read-only later).